datasets/greek_legal_code/README.md

```diff
@@ -0,0 +1,195 @@
+
+
+### Supported Tasks and Leaderboards
+
+The dataset supports:
+
+**Multi-class Text Classification:** Given the text of a document, a model predicts the corresponding class.
+
+**Few-shot and Zero-shot learning:** As already noted, the classes can be divided into three groups: frequent, few-shot, and zero- shot, depending on whether they were assigned to more than 10, fewer than 10 but at least one, or no training documents, respectively.
+
+| Level | Total | Frequent | Few-Shot (<10) | Zero-Shot |
+|---|---|---|---|---|
+|Volume|47|47|0|0|
+|Chapter|389|333|53|3|
+|Subject|2285|712|1431|142|
+
+### Languages
+
+All documents are written in Greek.
+
+## Dataset Structure
+
+### Data Instances
+
+
+```json
+{
+  "text": "179. ΑΠΟΦΑΣΗ ΥΠΟΥΡΓΟΥ ΜΕΤΑΦΟΡΩΝ ΚΑΙ ΕΠΙΚΟΙΝΩΝΙΩΝ Αριθ. Β-οικ. 68425/4765 της 2/17 Νοεμ. 2000 (ΦΕΚ Β΄ 1404) Τροποποίηση της 42000/2030/81 κοιν. απόφασης του Υπουργού Συγκοινωνιών «Κωδικοποίηση και συμπλήρωση καν. Αποφάσεων» που εκδόθηκαν κατ’ εξουσιοδότηση του Ν.Δ. 102/73 «περί οργανώσεως των δια λεωφορείων αυτοκινήτων εκτελουμένων επιβατικών συγκοινωνιών». ",
+  "volume": "ΣΥΓΚΟΙΝΩΝΙΕΣ",
+  "chapter": "ΛΕΩΦΟΡΕΙΑ",
+  "subject": "ΚΟΙΝΑ ΤΑΜΕΙΑ ΕΚΜΕΤΑΛΛΕΥΣΕΩΣ ΛΕΩΦΟΡΕΙΩΝ (Κ.Τ.Ε.Λ.)"
+}
+```
+
+### Data Fields
+
+The following data fields are provided for documents (`train`, `dev`, `test`):
+
+`text`: (**str**)  The full content of each document, which is represented by its `header` and `articles` (i.e., the `main_body`).\
+`volume`: (**str**)  The volume-level class it belongs to.\
+`chapter`: (**str**)  The chapter-level class it belongs to.\
+`subject`: (**str**)  The subject-level class it belongs to.
+
+### Data Splits
+
+| Split         | No of Documents                         | Avg. words |
+| ------------------- | ------------------------------------  |  --- |
+| Train | 28,536 | 600 |
+|Development | 9,511 | 574 |
+|Test | 9,516 | 595 |
+
+## Dataset Creation
+
+### Curation Rationale
+
+The dataset was curated by Papaloukas et al. (2021) with the hope to support and encourage further research in NLP for the Greek language.
+
+### Source Data
+
+#### Initial Data Collection and Normalization
+
+The ``Permanent Greek Legislation Code - Raptarchis'' is a thorough catalogue of Greek legislation since the creation of the Greek state in 1834 until 2015. It includes Laws, Royal and Presidential Decrees, Regulations and Decisions, retrieved from the Official Government Gazette, where Greek legislation is published. This collection is one of the official, publicly available sources of classified Greek legislation suitable for classification tasks.
```

Comment:
```suggestion
The ``Permanent Greek Legislation Code - Raptarchis`` is a thorough catalogue of Greek legislation since the creation of the Greek state in 1834 until 2015. It includes Laws, Royal and Presidential Decrees, Regulations and Decisions, retrieved from the Official Government Gazette, where Greek legislation is published. This collection is one of the official, publicly available sources of classified Greek legislation suitable for classification tasks.
```